src/ansible_compat/runtime.py

```diff
@@ -199,6 +203,31 @@ def install_collection(
             _logger.error(msg)
             raise InvalidPrerequisiteError(msg)
 
+    def install_collection_from_disk(
+        self, path: str, destination: Optional[Union[str, pathlib.Path]] = None
+    ) -> None:
+        """Build and install collection from a given disk path."""
+        if self.version_in_range(upper="2.11"):
+            # older versions of ansible able unable to install without building
+            with tempfile.TemporaryDirectory() as tmp_dir:
+                cmd = [
+                    "ansible-galaxy",
+                    "collection",
+                    "build",
+                    "--output-path",
+                    tmp_dir,
```

Comment:
The value should reside next to the argument
```suggestion
                    "--output-path", tmp_dir,
```

Reply:
Nope, black will reformat.

Reply:
Then add ignores. It's ridiculous when a formatter forces keeping tightly coupled things in different places.